cmd/docker/main.go

```diff
@@ -68,18 +83,18 @@ func Usage() {
 
 func main() {
 	// Get flags
-	cli, ok := handleFlags()
+	ok := handleFlags()
 
 	if !ok {
 		os.Exit(1)
 	}
 
-	if err := vicbackends.Init(cli.portLayerAddr, productName, &vchConfig, vchConfig.InsecureRegistries); err != nil {
+	if err := vicbackends.Init(*cli.portLayerAddr, productName, &vchConfig, vchConfig.InsecureRegistries); err != nil {
 		log.Fatalf("failed to initialize backend: %s", err)
 	}
 
 	// Start API server wit options from command line args
-	api := startServerWithOptions(cli)
+	api := startServer()
```

Comment:
previous pattern was better. Using globals for config is evil. It will complicate unit testing.